src/borrowing/shared.md

```diff
@@ -62,4 +62,4 @@ Notes on stack returns:
 
 </details>
 
-[Playground]: https://play.rust-lang.org/
+[Playground]: https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=0cb13be1c05d7e3446686ad9947c4671
```

Comment:
Just in case you haven't thought of it: we don't have a good way to maintaining these links as we modify the code. So I've been resisting adding them so far.

But let's see how it goes: perhaps we won't ever modify this and then the link is great :smile:

Reply:
> maybe we could automatically populate these links.

It's possible to embed the code directly into the link: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&code=fn+main%28%29+%7B%0A++++let+x+%3D+42%3B%0A%7D

So one could in principle (assuming the server accepts very long URLs) make a `mdbook` plugin which constructs such URLs for us automatically.